appveyor.yml

```diff
@@ -2,10 +2,6 @@ version: 6.0.0-alpha.18-{build}
 
 image: Visual Studio 2015
 
-cache:
-  - '%LocalAppData%\Microsoft\dotnet'
-  - '%HOMEDRIVE%%HOMEPATH%\.nuget\packages'
```

Comment:
As I mentioned in the description:
> Note that the `-cache` option is removed from appveyor.yml, but it probably should be added back after this PR is merged.

I planned to added it back after this PR gets merged, and add one more directory path: `%HOMEDRIVE%%HOMEPATH%\.dotnet`. `dotnet-cli` 2.0.0 depends on such a folder now.